best-practices-metadata-and-extension.md

```diff
@@ -87,13 +79,22 @@ Each asset that points to measurements should provide the fields defined below.
   
     `proj:code` can be set to `null` if one of the other two fields is provided.
     `proj:wkt2` and `proj:projjson` should never be set to `null`.
+
+    Typically specified at the Item level. If the projection is different
+    for all assets it should not be provided as an Item property. If most assets are one projection, and there is 
+    a single reprojected version (such as a Web Mercator preview image), it is sensible to specify the main projection in the 
+    Item and the alternate projection for the affected asset(s).
+
   - Two of the following: `proj:shape`, `proj:transform`, `proj:bbox`.
   
-    `proj:bbox` should be omitted if the values match `bbox` property exactly (i.e. if `proj:code` "is a geographic coordinate reference system, using the World Geodetic System 1984 (WGS 84) datum, with longitude and latitude units of decimal degrees" (see [RFC 7946, ch. 4](https://datatracker.ietf.org/doc/html/rfc7946#section-4)).
+    `proj:bbox` should be omitted if the values match `bbox` property exactly (i.e. if `proj:code` "is a geographic coordinate reference system, using the World Geodetic System 1984 (WGS 84) datum, with longitude and latitude units of decimal degrees" (see [RFC 7946, ch. 4](https://datatracker.ietf.org/doc/html/rfc7946#section-4))).
+
+    Typically specified at the Item level. If assets have different spatial resolutions and slightly different exact bounding boxes,
+    specify these per asset to indicate the size of the asset in pixels and its exact GeoTransform in the native projection.
   
 - **Raster Extension** (v2.x)
 
-  - `raster:spatial_resolution` (if multiple resolutions are provided in the item, otherwise specify it in the Item)
+  - `raster:spatial_resolution` (if multiple resolutions are provided in the Asset, otherwise in the Item)
   - `raster:scale`
   - `raster:offset`
```

Comment:
These are usually used on the asset level, not the Item property level. Thus I find the lack of distinction between Asset and Item a bit confusing.